packages/alfa-dom/src/node/element.ts

```diff
@@ -32,15 +33,17 @@ export class Element<N extends string = string>
     name: N,
     attributes: Iterable<Attribute> = [],
     children: Iterable<Node> = [],
-    style: Option<Block> = None
+    style: Option<Block> = None,
+    rectangle: Option<Rectangle> = None
```

Comment:
**Note:** we start having a lot of optional arguments on that function and it might be worth adding (or even replacing) an API with an `options` (or `extra`) object to selectively pass the ones we want.
Probably not for now, anyway.